### PR TITLE
#gh10718 added Frame Agreement and Order Call doctypes for SO

### PR DESCRIPTION
**Pull request by adi-stefan**
https://github.com/metasfresh/metasfresh/issues/10718

### CODE REVIEW

File: backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/c_orderline_id_with_missing_shipmentschedule_v.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+CREATE OR REPLACE VIEW c_orderline_id_with_missing_shipmentschedule_v AS
```
Comment (metas-ts) [inline backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/c_orderline_id_with_missing_shipmentschedule_v.sql:1]: please update the one we have (`backend/de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/ddl/views/C_OrderLine_ID_With_Missing_ShipmentSchedule_v.sql`)

File: backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java [hunk 2]
```diff
@@ -1600,7 +1600,8 @@ private DocStatus completeIt0()
 		//
 		// Offers
 		if (X_C_DocType.DOCSUBTYPE_Proposal.equals(docSubType)
-				|| X_C_DocType.DOCSUBTYPE_Quotation.equals(docSubType))
+				|| X_C_DocType.DOCSUBTYPE_Quotation.equals(docSubType)
+				|| X_C_DocType.DOCSUBTYPE_Rahmenvertrag.equals(docSubType))
```
Comment (metas-ts) [inline backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java:1604]: please chreate the new doctypes such that we have here there EN names, (like "Quotation")

**Review by metas-ts — CHANGES_REQUESTED** [review]

File: backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java (hunk 2 above)
Comment (metas-ts) [reply backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java:1604]: So, do you have in here the SQL to deactivate the doc-types? I didn't see it, but in this large SQL file i might have overlooked it

Comment (adi-stefan) [reply backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java:1604]: Yep, I've changed the script to create the docTypes with IsActive='N'

File: backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/c_orderline_id_with_missing_shipmentschedule_v.sql (hunk 1 above)
Comment (adi-stefan) [reply backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/c_orderline_id_with_missing_shipmentschedule_v.sql:1]: Missed that. Fixed.

File: backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java (hunk 2 above)
Comment (adi-stefan) [reply backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java:1604]: >please chreate the new doctypes such that we have here there EN names, (like "Quotation")

Done.

**Review by metas-ts — APPROVED** [review]

**Pull request merged**